types/npm-package-json-lint/index.d.ts

```diff
@@ -0,0 +1,53 @@
+// Type definitions for npm-package-json-lint 5.1
+// Project: https://github.com/tclindner/npm-package-json-lint
+// Definitions by: Hannes Leutloff <https://github.com/yeldiRium>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.3
+
+interface LintIssue {
+    lintId: string;
+    severity: 'error' | 'warning';
+    node: 'string';
+    lintMessage: 'string';
+}
+
+interface FileLintResult {
+    filePath: string;
+    issues: LintIssue[];
+    ignored: boolean;
+    errorCount: number;
+    warningCount: number;
+}
+
+interface LinterResult {
+    results: FileLintResult[];
+    ignoreCount: number;
+    errorCount: number;
+    warningCount: number;
+}
+
+type NpmPackageJsonLintOptions = {
+    cwd?: string;
+    config?: object;
+    configFile: string;
+    configBaseDirectory?: string;
+    quiet?: boolean;
+    ignorePath?: string;
+    fix?: boolean;
+} & (
+    | {
+          packageJsonObject: object;
+          packageJsonFilePath?: string;
+      }
+    | {
+          patterns: string[];
+      }
+);
+
+declare class NpmPackageJsonLint {
+    constructor(options: NpmPackageJsonLintOptions);
+
+    lint(): LinterResult;
+}
+
+export { FileLintResult, LinterResult, LintIssue, NpmPackageJsonLint };
```

Comment:
this looks wrong, this is CJS module, plain old CJS:
```suggestion
export = NpmPackageJsonLint;
```
https://github.com/tclindner/npm-package-json-lint/blob/master/src/NpmPackageJsonLint.js#L174
you would need to e.g. add a namespace to wrap interfaces, so they are exported:
```ts
declare namespace NpmPackageJsonLint {
}
```

Reply:
disregard, found wrapper:
https://unpkg.com/npm-package-json-lint@5.1.0/src/api.js
https://unpkg.com/browse/npm-package-json-lint@5.1.0/package.json